src/validators/duplicate_stops.rs

```diff
@@ -0,0 +1,91 @@
+use crate::validators::issues::*;
+use geo::algorithm::haversine_distance::HaversineDistance;
+use geo::Point;
+use itertools::Itertools;
+
+pub fn validate(gtfs: &gtfs_structures::Gtfs) -> Vec<Issue> {
+    let mut issues = vec![];
+
+    for (stop_a, stop_b) in gtfs.stops.values().tuple_combinations() {
```

Comment:
this way of comparing stops is very costly (you perform nb_stops² comparisons).

you can improve this by using a `BTreeMap` or `HashMap` with a complex key. For the distance check, it's a bit trickier, you can either use a [rtree](https://en.wikipedia.org/wiki/R*_tree) or maybe [geohashes](https://en.wikipedia.org/wiki/Geohash) ?

Reply:
to test if that is a performance problem, can you validate the Netherlands gtfs ?

Reply:
Just tested with the Ile-de-France Mobilités gtfs, it wasn't even a matter of hundredth of a second so I guess it's not a performance problem. But thank you for the heads up =)